### PR TITLE
Various fixes for issues found by the audit

### PR DESCRIPTION
**Pull request by WhatisRT**
# Description

There were smaller changes that I made while responding to the audit.

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [x] Code is formatted according to [CONTRIBUTING.md](https://github.com/input-output-hk/formal-ledger-specifications/blob/master/CONTRIBUTING.md)
- [x] Self-reviewed the diff

### CODE REVIEW

File: src/Ledger/Enact.lagda [hunk 1]
```diff
@@ -17,8 +17,11 @@ open import Ledger.GovernanceActions gs
 Figure~\ref{fig:enact-defs} contains some definitions required to
 define the ENACT transition system. \EnactEnv is the environment and
 \EnactState the state of ENACT, which enacts a governance action. All
-governance actions except \TreasuryWdrl and \Info modify \EnactState, which of course can have further
-consequences. Also, enacting these governance actions is the
+governance actions except \TreasuryWdrl and \Info modify \EnactState
+permanently, which of course can have further
+consequences. \TreasuryWdrl accumulates withdrawal temporarily in \EnactState
```
Comment (williamdemeo) [inline src/Ledger/Enact.lagda:22]:
```suggestion
consequences. \TreasuryWdrl accumulates withdrawal temporarily in \EnactState,
```

File: src/Ledger/Gov.lagda [hunk 2]
```diff
@@ -269,7 +270,11 @@ Figure~\ref{defs:gov-rules}.
 
 For \GOVVote, we check that the governance action being voted on
 exists and the role is allowed to vote. \canVote is defined in
-Figure~\ref{fig:ratification-requirements}.
+Figure~\ref{fig:ratification-requirements}. Note that there are no
+checks on whether the credential is actually associated with the
+role. This means that anyone can vote for e.g. the \CC role. However,
```
Comment (williamdemeo) [inline src/Ledger/Gov.lagda:275]:
```suggestion
role. This means that anyone can vote for, e.g., the \CC role. However,
```

File: src/Ledger/Notation.lagda [hunk 3]
```diff
@@ -28,4 +28,7 @@ In this section, we introduce some notations used in this document.
   \AgdaBound{m}~\AgdaFunction{∪ˡ}~\AgdaBound{m'} for their left-biased union. This means that
   key-value pairs in \AgdaBound{m} are guaranteed to be in the union, while key-value pairs in
   \AgdaBound{m'} will be in the union if and only if the keys don't collide.
+\item[Map addition] For maps \AgdaBound{m} and \AgdaBound{m'}, we write
+  \AgdaBound{m}~\AgdaFunction{∪⁺}~\AgdaBound{m'} for their union where keys that appear
```
Comment (williamdemeo) [inline src/Ledger/Notation.lagda:32]:
```suggestion
  \AgdaBound{m}~\AgdaFunction{∪⁺}~\AgdaBound{m'} for their union, where keys that appear
```

**Review by williamdemeo — CHANGES_REQUESTED** [review]
I've suggested a couple of minor changes, but feel free to ignore these changes.  More importantly, the latest master needs to be merged into this branch and conflicts must be resolved.  (I merged the latest master into my local checkout of this branch and reconciled the conflicts in a way that seemed most reasonable to me.  I could push that to this branch, but it's probably safer for you to resolve the conflicts yourself.  The main non-trivial conflict is that master now defines a `keyDeposit` parameter which is used instead of `poolDeposits` in the `Deleg` module.

**Comment by WhatisRT** [comment]
@williamdemeo I've incorporated your suggestions, but you still need to approve the PR

**Comment by WhatisRT** [comment]
@williamdemeo I'm now using my admin powers to merge this PR anyway. Your message sounds like you're fine with merging it, so you should have approved this PR in the UI.

**Pull request merged**